playground/tailwind/package.json

```diff
@@ -15,7 +15,6 @@
     "vue-router": "^4.0.15"
   },
   "devDependencies": {
-    "@vitejs/plugin-vue": "workspace:*",
-    "ts-node": "^10.7.0"
```

Comment:
The `ts-node` here is intentional to test loading `postcss.config.ts`, which failed in the past